### ISSUE TITLE
(Debian 11) install-timelord.sh fails

### ISSUE THREAD

**Issue by hajes** [Chia-Network/chia-blockchain#9998]
### What happened?

it looks like script `./install-timelord.sh` fails to recognize Debian Linux version, uninstalls `cmake` and then whines about missing `cmake`.

Kludge is to comment out `ubuntu_cmake_install` inside the script, `apt install cmake` and run the script again.

...
else
        if [ -e venv/bin/python ] && test $UBUNTU_DEBIAN; then
                echo "Installing chiavdf from source on Ubuntu/Debian"
                # If Ubuntu version is older than 20.04LTS then upgrade CMake
`#               ubuntu_cmake_install`
                # Install remaining needed development tools - assumes venv and prior run of install.sh
                echo apt-get install libgmp-dev libboost-python-dev lib"$PYTHON_VERSION"-dev libboost-system-dev build->                sudo apt-get install libgmp-dev libboost-python-dev lib"$PYTHON_VERSION"-dev libboost-system-dev build->                echo venv/bin/python -m pip install --force --no-binary chiavdf "$CHIAVDF_VERSION"
                venv/bin/python -m pip install --force --no-binary chiavdf "$CHIAVDF_VERSION"
                symlink_vdf_bench "$PYTHON_VERSION"
...

### Version

1.2.11

### What platform are you using?

Linux

### What ui mode are you using?

CLI

### Relevant log output

```shell
./install-timelord.sh
Timelord requires CMake 3.14+ to compile vdf_client.
Python version: python3.9
apt-get is /usr/bin/apt-get
Found Ubuntu/Debian.
Installing chiavdf from source on Ubuntu/Debian
Ubuntu version is pre 20.04LTS - installing CMake with snap.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
snap is already the newest version (2013-11-29-11).
0 upgraded, 0 newly installed, 0 to remove and 15 not upgraded.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following packages were automatically installed and are no longer required:
  cmake-data libarchive13 libjsoncpp24 librhash0
Use 'sudo apt autoremove' to remove them.
The following packages will be REMOVED:
  cmake*
0 upgraded, 0 newly installed, 1 to remove and 15 not upgraded.
After this operation, 23.5 MB disk space will be freed.
(Reading database ... 87093 files and directories currently installed.)
Removing cmake (3.18.4-2+deb11u1) ...
Processing triggers for man-db (2.9.4-2) ...
sudo: snap: command not found
apt-get install libgmp-dev libboost-python-dev libpython3.9-dev libboost-system-dev build-essential -y
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
libboost-python-dev is already the newest version (1.74.0.3).
libboost-system-dev is already the newest version (1.74.0.3).
build-essential is already the newest version (12.9).
libgmp-dev is already the newest version (2:6.2.1+dfsg-1+deb11u1).
libpython3.9-dev is already the newest version (3.9.2-1).
The following packages were automatically installed and are no longer required:
  cmake-data libarchive13 libjsoncpp24 librhash0
Use 'sudo apt autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 15 not upgraded.
venv/bin/python -m pip install --force --no-binary chiavdf chiavdf==1.0.3
Collecting chiavdf==1.0.3
  Using cached chiavdf-1.0.3.tar.gz (635 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Building wheels for collected packages: chiavdf
  Building wheel for chiavdf (pyproject.toml) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/hajes/chia-blockchain/venv/bin/python /home/hajes/chia-blockchain/venv/lib/python3.9/site-packages/pip/_vendor/pep517/in_process/_in_process.py build_wheel /tmp/tmp32sreh4d
       cwd: /tmp/pip-install-nvyzium3/chiavdf_4eb51f5c8feb48009a1e4d69db5c5950
  Complete output (58 lines):
  running bdist_wheel
  running build
  running build_ext
  Traceback (most recent call last):
    File "setup.py", line 96, in run
      out = subprocess.check_output(["cmake", "--version"])
    File "/usr/lib/python3.9/subprocess.py", line 424, in check_output
      return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
    File "/usr/lib/python3.9/subprocess.py", line 505, in run
      with Popen(*popenargs, **kwargs) as process:
    File "/usr/lib/python3.9/subprocess.py", line 951, in __init__
      self._execute_child(args, executable, preexec_fn, close_fds,
    File "/usr/lib/python3.9/subprocess.py", line 1823, in _execute_child
      raise child_exception_type(errno_num, err_msg, err_filename)
  NotADirectoryError: [Errno 20] Not a directory: 'cmake'

  During handling of the above exception, another exception occurred:

  Traceback (most recent call last):
    File "~/chia-blockchain/venv/lib/python3.9/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 363, in <module>
      main()
    File "/home/hajes/chia-blockchain/venv/lib/python3.9/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 345, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "/home/hajes/chia-blockchain/venv/lib/python3.9/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 261, in build_wheel
      return _build_backend().build_wheel(wheel_directory, config_settings,
    File "/tmp/pip-build-env-n_etiw_9/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 230, in build_wheel
      return self._build_with_temp_dir(['bdist_wheel'], '.whl',
    File "/tmp/pip-build-env-n_etiw_9/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 215, in _build_with_temp_dir
      self.run_setup()
    File "/tmp/pip-build-env-n_etiw_9/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 158, in run_setup
      exec(compile(code, __file__, 'exec'), locals())
    File "setup.py", line 267, in <module>
      setup(
    File "/tmp/pip-build-env-n_etiw_9/overlay/lib/python3.9/site-packages/setuptools/__init__.py", line 155, in setup
      return distutils.core.setup(**attrs)
    File "/tmp/pip-build-env-n_etiw_9/overlay/lib/python3.9/site-packages/setuptools/_distutils/core.py", line 148, in setup
      return run_commands(dist)
    File "/tmp/pip-build-env-n_etiw_9/overlay/lib/python3.9/site-packages/setuptools/_distutils/core.py", line 163, in run_commands
      dist.run_commands()
    File "/tmp/pip-build-env-n_etiw_9/overlay/lib/python3.9/site-packages/setuptools/_distutils/dist.py", line 967, in run_commands
      self.run_command(cmd)
    File "/tmp/pip-build-env-n_etiw_9/overlay/lib/python3.9/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
      cmd_obj.run()
    File "/tmp/pip-build-env-n_etiw_9/overlay/lib/python3.9/site-packages/wheel/bdist_wheel.py", line 299, in run
      self.run_command('build')
    File "/tmp/pip-build-env-n_etiw_9/overlay/lib/python3.9/site-packages/setuptools/_distutils/cmd.py", line 313, in run_command
      self.distribution.run_command(command)
    File "/tmp/pip-build-env-n_etiw_9/overlay/lib/python3.9/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
      cmd_obj.run()
    File "/tmp/pip-build-env-n_etiw_9/overlay/lib/python3.9/site-packages/setuptools/_distutils/command/build.py", line 135, in run
      self.run_command(cmd_name)
    File "/tmp/pip-build-env-n_etiw_9/overlay/lib/python3.9/site-packages/setuptools/_distutils/cmd.py", line 313, in run_command
      self.distribution.run_command(command)
    File "/tmp/pip-build-env-n_etiw_9/overlay/lib/python3.9/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
      cmd_obj.run()
    File "setup.py", line 98, in run
      raise RuntimeError(
  RuntimeError: CMake must be installed to build the following extensions: chiavdf
  ----------------------------------------
  ERROR: Failed building wheel for chiavdf
Failed to build chiavdf
ERROR: Could not build wheels for chiavdf, which is required to install pyproject.toml-based projects
```

**Comment by esaung** [comment]
Amine, can you take a look at this issue?

**Comment by AmineKhaldi** [comment]
@hajes could you please see if PR #10841 fixes this for you? Thanks!

**Comment by emlowe** [comment]
Should be fixed in 1.3.4 - closing issue
